html/pfappserver/root/static.alt/src/components/pfProgressApi.vue

```diff
@@ -3,79 +3,44 @@
 </template>
 
 <script>
-import apiCall from '@/utils/api'
 import { createDebouncer } from 'promised-debounce'
 
 export default {
-  name: 'pf-progress-api',
+  name: 'pfProgressApi',
```

Comment:
Don't use CamelCase, keep the previous format. This way, we already have the proper component name format when injecting it in App.vue.